sections/introduction.md

```diff
@@ -0,0 +1,49 @@
+# Introduction
+
+> ℹ️ These docs are written for engineers and ad ops professionals looking to integrate zero.js in their site.
+
+## Background
+
+The programmatic advertising system emits worrying amounts of CO2. In the worst cases, delivering a single impression involves the processing, storing and transmitting of data across hundreds of nodes. The result of this noisy system is that a single impression can emit over 1 gram of carbon dioxide 🤯
+
+> ℹ️ Programmatic advertising falls into an Advertiser's *scope 3* (supply chain) emissions and so they must reduce these emissions to meet net zero targets (offsetting is insufficient).
+
+The Glimpse Zero system enables publishers to sell their inventory in a carbon efficient manner. This inventory is called a Green Media Product. In the next section we'll provide an overview for how we do this.
```

Comment:
I would change "sell" to "make their inventory available" they don't have to sell this they can